src/Tempest/Support/src/Arr/ManipulatesArray.php

```diff
@@ -20,7 +20,7 @@
 trait ManipulatesArray
 {
     /** @var array<TKey, TValue> */
-    private(set) array $value;
+    public private(set) array $value;
```

Comment:
`public private(set)` is the same as `private(set)`, as `private(set)` is a shorthand. What's your reasoning for adding it? Maybe to be more explicit? I don't necessarily mind it, but we need to figure out a consensus within the whole codebase.

Reply:
My reason is exactly what you mentioned - it being more explicit.

```php
class SomeClass
{
    function foo() // this is public by default
    {
    }
}

class SomeOtherClass
{
    public function bar() // but this reads much better with the explicit visibility keyword
    {
    }
}
```

Reply:
Nuked for now as per our Discord discussion with you and the rest of the contributors.